apps/docs/content/components/CheckboxGroup/react.mdx

```diff
@@ -133,7 +133,13 @@ When space is limited, checkboxes can be arranged horizontally using the [Stack]
   <CheckboxGroup.Caption>
     Some inline checkboxes with a visually hidden label
   </CheckboxGroup.Caption>
-  <Stack direction="horizontal" gap="normal" padding="none">
+  <Stack
+    direction="horizontal"
+    gap="normal"
+    padding="none"
+    flexWrap="wrap"
+    style={{rowGap: 'var(--base-size-8)'}}
```

Comment:
I think this is fine, but also quite ugly for our public docs. Could we remove it for now to keep it clean? We rarely have folks viewing docs on mobile, and there's bigger issues with that viewport generally with our docs.

Reply:
```suggestion
```

Reply:
No prob, I guessed you might say that so I added that change in an easily-revertible commit 😄

That's been removed now